docs/database.rst

```diff
@@ -8,7 +8,8 @@ This functionality should still be considered experimental!
 Mailu Postgresql
 ----------------
 
-Mailu optionally comes with a pre-configured Postgresql image.
+Mailu optionally comes with a pre-configured Postgresql image, which as of 1.8, is deprecated
+and will be removed in 1.9.
 This images has the following features:
```

Comment:
Since you are modifying database.rst anyway, could you fix this typo?
This image**s** has => This image has ...

Reply:
I'm referring to line number 13 in case this is not clear.